integration/configs/integrationTest/test_app_video_end2end_test.config.json

```diff
@@ -1,27 +1,25 @@
 {
-    "`": 1,
-    "name": "test_app_video_end2end_test.config.json %ts",
-    "grids": [
-    ],
-    "tests": [
-      {
-        "name": "Test App End2End Video Test %ts",
-        "description": "Test App End2End Video Test",
-        "tupleSize": 2,
-        "noOfThreads": 1,
-        "testImpl": "VideoEnd2EndTest.js",
-        "payload": {
-          "url": "http://localhost:8080/",
-          "port": 30002,
-          "retry": 2
-        }
+  "`": 1,
+  "name": "test_app_video_end2end_test.config.json %ts",
+  "grids": [],
+  "tests": [
+    {
+      "name": "Test App End2End Video Test %ts",
+      "description": "Test App End2End Video Test",
+      "tupleSize": 2,
```

Comment:
Why do we need tuple size 2 here, if were only doing chrome?

Reply:
This test is an exception. We should update its `tupleSize` to 1 as well to get rid of the socket.io stuff in our test framework, but it requires the rewriting of the `VideoEnd2EndTest.js` which uses the multi-session way to execute the test now. Since our Canary doesn't use this test at this moment, I didn't make the change to it at this time. But later we should update it.